apps/condo/domains/ticket/components/BaseTicketForm/index.tsx

```diff
@@ -276,15 +277,16 @@ export const TicketSourceSelect: React.FC = () => {
         <Form.Item
             label={TicketSourceLabel}
             required
-            name='source'
+            name={loading ? null : 'source'}
```

Comment:
It looks strange.
What the reason of doing this?

Reply:
if item have name, then it use value from form. But we need set custom value in `Input` component if loading data that's why name set null

Reply:
It's been refactored